marin/execution/executor.py

```diff
@@ -520,15 +520,19 @@ def __init__(
         self.refs: dict[ExecutorStep, ray.ObjectRef] = {}
         self.step_infos: list[ExecutorStepInfo] = []
         self.executor_info: ExecutorInfo | None = None
+        if not is_local_ray_cluster():
```

Comment:
Definitely much cleaner, sorry I missed this originally. Should I go through and rip out the head node special resource then? This + CPU:0 seems sufficient to prevent accidental head node scheduling and allow intentional head node scheduling.

Reply:
eh

Reply:
i guess probably